<launch>
    <node name="imageproc_node" pkg="imageproc_node" type="imageproc_node.py" output="screen">
        <rosparam param="device">/dev/ttySAC1</rosparam>
        <rosparam param="invertLeft">True</rosparam>
        <rosparam param="invertRight">True</rosparam>
        <rosparam param="minThrottle">-2400</rosparam>
        <rosparam param="maxThrottle">2400</rosparam>
        <rosparam param="linearGain">1200</rosparam>
        <rosparam param="angularGain">1200</rosparam>

    </node>
    <node name="estop" pkg="imageproc_node" type="emergencystop.py" output="screen" />
    <node name="heartbeat" pkg="imageproc_node" type="heartbeat.py" output="screen" />
</launch>

